riscv/Cargo.toml

```diff
@@ -24,20 +24,22 @@ powdr-pipeline.workspace = true
 powdr-riscv-executor.workspace = true
 powdr-riscv-syscalls.workspace = true
 
-goblin = { version = "0.8" }
+gimli = "0.31"
+goblin = "0.8"
 lazy_static = "1.4.0"
 itertools = "0.13"
 lalrpop-util = { version = "^0.19", features = ["lexer"] }
 log = "0.4.17"
 mktemp = "0.5.0"
 num-traits = "0.2.15"
 raki = "0.1.4"
-serde_json = "1.0"
 # This is only here to work around https://github.com/lalrpop/lalrpop/issues/750
 # It should be removed once that workaround is no longer needed.
 regex-syntax = { version = "0.6", default_features = false, features = [
     "unicode",
 ] }
+serde_json = "1.0"
+thiserror = "1.0"
```

Comment:
Considering there is no practical difference, since the more specific version requested in `backend` satisfy the version requested here, and there is no good reason to request the more specific version in this crate, I think it is fine as is.